GoogleUtilities/FileDataStorage/GULStorageHeartbeat.m

```diff
@@ -35,20 +36,61 @@ - (instancetype)initWithFileURL:(NSURL *)fileURL {
   return self;
 }
 
+- (nullable NSMutableDictionary *)getDictionary {
+  NSError *error;
+  NSData *objectData = [NSData dataWithContentsOfURL:self.fileURL options:0 error:&error];
```

Comment:
Just a reminder, we still need to synchronize this, `setHearbeatDate: forTag:` and "check and create directory"(that is to be moved here) methods  with [NSFileCoordinator](https://developer.apple.com/documentation/foundation/nsfilecoordinator?language=objc)